src/angularStorage/services/internalStore.js

```diff
@@ -1,8 +1,12 @@
 angular.module('angular-storage.internalStore', ['angular-storage.localStorage', 'angular-storage.sessionStorage'])
   .factory('InternalStore', function($log, $injector) {
 
-    function InternalStore(namespace, storage, delimiter) {
+    function InternalStore(namespace, storage, delimiter, useCache) {
       this.namespace = namespace || null;
+      if (typeof useCache === 'undefined' || useCache === undefined || useCache == null) {
```

Comment:
Oh, not again cast-to-bool operator :) 
Anyway, I don't think it would work. !! defaults null/undefined to false. I need to default it to true.

Or, I could revert the semantics and rename it to 'dontUseCache' Then the !! would work I think